From 301ca3beba18d92873fa8544be47a4e357214c10 Mon Sep 17 00:00:00 2001
From: Zhang Qiang <qiang.z.zhang@intel.com>
Date: Tue, 10 Dec 2013 15:37:49 +0800
Subject: [PATCH 10/10] download packages to different dirs for each build
 instances

This patch can avoid download error during multiple build instance
runing at the same time

Change-Id: Ie2ae7b6e73c8f7c7358b4dfd8b758ec662db7bbd
---
 init_buildsystem |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/init_buildsystem b/init_buildsystem
index 512ba1f..e72613a 100755
--- a/init_buildsystem
+++ b/init_buildsystem
@@ -534,7 +534,7 @@ downloadpkg()
 	cleanup_and_exit 1
     fi
 
-    local destdir="$cachedir/tmp"
+    local destdir="$cachedir/tmp_$$"
     mkdir -p "$destdir"
     echo "downloading $url ... ";
     $BUILD_DIR/download "$destdir" "$url" || cleanup_and_exit 1
@@ -554,6 +554,7 @@ downloadpkg()
 	;;
     esac
     mv "$destfile" "$SRC" || cleanup_and_exit 1
+    rm -rf $destdir
 }
 
 getcachedir()
-- 
1.7.9.5

